packages/cli/src/zosfiles/edit/Edit.utils.ts

```diff
@@ -0,0 +1,328 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Download, Upload, IZosFilesResponse, IDownloadOptions, IUploadOptions } from "@zowe/zos-files-for-zowe-sdk";
+import { AbstractSession, IHandlerParameters, ImperativeError, ProcessUtils, GuiResult,
+    TextUtils, IDiffOptions } from "@zowe/imperative";
+import { CompareBaseHelper } from "../compare/CompareBaseHelper";
+import { CliUtils } from "@zowe/imperative";
+import { existsSync, unlinkSync } from "fs";
+import { tmpdir } from "os";
+import path = require("path");
+import LocalfileDatasetHandler from "../compare/lf-ds/LocalfileDataset.handler";
+import LocalfileUssHandler from "../compare/lf-uss/LocalfileUss.handler";
+
+
+/**
+ * enum of prompts to be used as input to {@link EditUtilities.promptUser} during the file editing process
+ * @export
+ * @enum
+ */
+export enum Prompt {
+    useStash,
+    doneEditing,
+    continueToUpload,
+    viewUpdatedRemote
+}
+
+/**
+ * An enum to supply more structure within {@link LocalFile}
+ * @export
+ * @enum
+ */
+export enum EditFileType {
+    uss = 'uss',
+    ds = 'ds'
+}
```

Comment:
Since there are only 2 possible values that are both string literals, a union type might be a better fit than an `enum`. For example: `export type EditFileType = "uss" | "ds";`